pkg/planner/core/explain.go

```diff
@@ -271,38 +271,55 @@ func (p *PhysicalTableScan) OperatorInfo(normalized bool) string {
 			buffer.WriteString(runtimeFilter.ExplainInfo(false))
 		}
 	}
+
+	var annIndexes []string
+	var invertedIndexes []string
```

Comment:
It is recommended to set a predefined size. Even if the possible size is unknown, there should be a preset size. The suggested size is 4.